packages/venia-concept/src/components/Pagination/pagination.css

```diff
@@ -0,0 +1,33 @@
+.root {
+    display: grid;
+    grid-auto-columns: 1.8rem;
```

Comment:
This is kind of a nitpick, but in general we try to do things on a `4px/0.25rem` or `8px/0.5rem` grid. Any way we can adjust these down to `1.75rem` or up to `2rem`?

Reply:
Shouldn't be a problem, I'll check to make sure everything still aligns properly.